config/config_koanf.go

```diff
@@ -0,0 +1,97 @@
+package config
+
+import (
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/knadh/koanf"
+	"github.com/knadh/koanf/parsers/yaml"
+	"github.com/knadh/koanf/providers/env"
+	"github.com/knadh/koanf/providers/file"
+	"github.com/mitchellh/mapstructure"
+)
+
+const prefix = "BLOCKY_"
```

Comment:
Maybe we should consolidate all config related constants? Having all of that in this package would make sense to me, but it's fine to leave as is too.  
I'm thinking of the ones in `root.go`: `defaultConfigPath`, `configFileEnvVar`, `configFileEnvVarOld`.

Reply:
Yeah I think you're right.
All environment related stuff in the same go file with comments would be best.
Currently it's all config related so the config package would be reasonable.🤔